installing/README.md

```diff
@@ -0,0 +1,10 @@
+# Installing Nix
```

Comment:
OK, will do but not in this branch. I think that part should then probably go into the main README.md which is the first thing users see.